pandas/plotting/_core.py

```diff
@@ -2704,18 +2704,56 @@ def __call__(self, x=None, y=None, kind='line', ax=None,
 
     def line(self, x=None, y=None, **kwds):
         """
-        Line plot
+        Plot `DataFrame` columns as lines.
+
+        This function is useful to plot lines using `DataFrame`'s values
+        as coordinates.
+        It allows to specify which columns to use for the axes.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : int, str, list of int or list of str, optional
+            It defines the X axis coordinates.
+            Either the location or the label of the columns to be used.
+            By default, it will use the `DataFrame` indices.
+        y : int, str, list of int or list of str, optional
+            It defines the values to be plotted.
+            Either the location or the label of the columns to be used.
+            By default, it will use the remaining `DataFrame` columns.
+        kwds : optional
             Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
 
         Returns
         -------
-        axes : matplotlib.AxesSubplot or np.array of them
+        axes : :class:`matplotlib.AxesSubplot` or :meth:`np.array` of
```

Comment:
:meth:`np.array` -> ndarray. I don't care either way about the sphinx markup so either

    :class:numpy.ndarray

or

    numpy.ndarray